### PR TITLE
Support parameter path - llcsharp

### PR DESCRIPTION
**Pull request by shawncx**
For llcsharp part, support parameter path.
Still need powershell-v2

### CODE REVIEW

File: powershell/plugins/plugin-create-inline-properties.ts [hunk 1]
```diff
@@ -363,16 +364,12 @@ async function createVirtuals(state: State): Promise<PwshModel> {
  
     Individual models can change the $THRESHOLD for generate
   */
-  // dolauli this plugin is used to expand the property in an object.
-  // dolauli inline-threshold could be set in readme.md
-  // skip-for-time-being
-  const threshold = await state.getValue('inlining-threshold', 24);
+  // const threshold = await state.getValue('inlining-threshold', 24);
+  const threshold = 24;
```
Comment (dolauli) [inline powershell/plugins/plugin-create-inline-properties.ts:368]: Any special reason to fix the threshold to 24?

File: powershell/plugins/plugin-tweak-model.ts [hunk 2]
```diff
@@ -75,86 +75,97 @@ async function tweakModelV2(state: State): Promise<PwshModel> {
...
+
+      // sub class
+      if (objSchema.discriminatorValue) {
+        objSchema.language.default.discriminatorValue = objSchema.extensions?.['x-ms-discriminator-value'];
+      }
     }
   }
 
-  if (await state.getValue('azure', false)) {
-    universalId.addProperty(new Property('id', '', new StringSchema('_identity_type_', 'Resource identity path'),
```
Comment (dolauli) [inline powershell/plugins/plugin-tweak-model.ts:110]: We need to keep this part to support viaIdentity for azure

File: powershell/cmdlets/namespace.ts [hunk 3]
```diff
@@ -56,7 +56,7 @@ export class NewCmdletNamespace extends Namespace {
         continue;
       }
       // skip-for-time-being
-      this.addClass(await new NewCmdletClass(this, operation, this.state.path('commands', 'operations', index)).init());
+      // this.addClass(await new NewCmdletClass(this, operation, this.state.path('commands', 'operations', index)).init());
```
Comment (dolauli) [inline powershell/cmdlets/namespace.ts:59]: Any reason for commenting out this line

File: powershell/plugins/llcsharp.ts [hunk 4]
```diff
@@ -8,14 +8,25 @@ import { join } from 'path';
...
+  const a = applyOverrides(content, overrides);
+  return a;
+}
+
 export async function llcsharp(service: Host) {
   try {
     const project = await new Project(service).init();
 
-    await project.writeFiles(async (fname, content) => service.WriteFile(join(project.apifolder, fname), applyOverrides(content, project.overrides), undefined, 'source-file-csharp'));
+    // await project.writeFiles(async (fname, content) => service.WriteFile(join(project.apifolder, fname), test(content, project.overrides), undefined, 'source-file-csharp'));
```
Comment (dolauli) [inline powershell/plugins/llcsharp.ts:28]: It is better we could align with other writeFiles function calls

File: powershell/plugins/plugin-create-inline-properties.ts (hunk 1 above)
Comment (shawncx) [reply powershell/plugins/plugin-create-inline-properties.ts:368]: revert the change. Thanks

File: powershell/plugins/plugin-tweak-model.ts (hunk 2 above)
Comment (shawncx) [reply powershell/plugins/plugin-tweak-model.ts:110]: Got it. Add it back

File: powershell/cmdlets/namespace.ts (hunk 3 above)
Comment (shawncx) [reply powershell/cmdlets/namespace.ts:59]: Consider powershell-v2 has some issue

File: powershell/plugins/llcsharp.ts (hunk 4 above)
Comment (shawncx) [reply powershell/plugins/llcsharp.ts]: just for test. Add it back

File: powershell/cmdlets/namespace.ts (hunk 3 above)
Comment (shawncx) [reply powershell/cmdlets/namespace.ts]: Fixed. And also change pipeline to use Powershellv2. Let's confirm whether it can pass the test ;-)

**Pull request merged**